src/handlers.ts

```diff
@@ -85,14 +86,11 @@ export async function handleChromiumCheck(): Promise<void> {
     const latestUpstreamVersion = upstreamVersions[upstreamVersions.length - 1];
     if (compareVersions(latestUpstreamVersion, chromiumVersion) > 0) {
       d(`branch ${branch.name} could upgrade from ${chromiumVersion} to ${latestUpstreamVersion}`);
-      const forkBranchName = await rollChromium4(branch.name, latestUpstreamVersion);
-      if (forkBranchName) {
-        d(`fetching chromium commits in ${chromiumVersion}..${latestUpstreamVersion}`);
-        const chromiumCommits = await getChromiumCommits(chromiumVersion, latestUpstreamVersion);
-        d('raising PR');
-        await raisePR4(forkBranchName, branch.name, chromiumCommits, chromiumVersion, latestUpstreamVersion, false);
-      } else {
-        d('chromium upgrade failed, not raising a PR');
+
+      try {
+        await rollChromium4(branch, latestUpstreamVersion);
+      } catch (e) {
+        d(`Error rolling ${branch.name} to ${latestUpstreamVersion}: ${e}`);
```

Comment:
Can we add `e` as a second param instead of stringifying it.  If we do it like this we lose the stacktrace.

![image](https://user-images.githubusercontent.com/6634592/57875097-ef16f580-77c6-11e9-9a42-23e7249ebec9.png)

Reply:
ah i was looking for the debug docs that this worked but it didn't list anything. done 👍